tests/conftest.py

```diff
@@ -37,5 +58,14 @@ def dbt_profile_target():
 
 @pytest.fixture(scope="class")
 def unique_schema(request, prefix) -> str:
-    unique_schema = f"dbt_test_{prefix}"
-    return unique_schema
\ No newline at end of file
+    unique_schema = f"dbt_test_{prefix}"   
+    return unique_schema
+
+@pytest.fixture(scope="class")
```

Comment:
What is the need of overriding prefix method specially in this release?

Reply:
We needed to override this because we have some clause in ci-integration file where we run the tests again with existing db DBT_TEST, and at the same time we needed to main the length of schema name for grants test, as there is constraint for it in DBC.allrights view.

Reply:
Had a meeting with @VarunSharma15  and got understand of why this prefix is overridden. the basic difference with this implementation is removed **_test__** from prefix to accommodate schema name within limit of 32 characters.